src/test/resources/features/highlighting/inspections/unresolve_element.feature

```diff
@@ -153,3 +153,18 @@ Feature: Inspection: Unresolved element
      Unresolved reference '{2}/boo.yaml'
      """
     When I check highlighting warnings
+
+  Scenario Outline: Know objects not marked as unresolved
```

Comment:
Better to rename the test to: Implicit variables not marked as unresolved.

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/424/commits/dcaea2a9a74dff8d433706ce1fff2d61620318b4